src/app/features/contentful/services/contentful/contentful.service.ts

```
@@ -1,52 +1,51 @@
 
-export const INITIAL_ENTRIES = () => ({
-	exprience: [],
-	skill: [],
-	company: [],
-	school: [],
-	diploma: [],
-	training: [],
-});
+export const INITIAL_ENTRIES = (): Record<EntryId, []> =>
+	Object.keys(entriesSchema.shape).reduce(
+		(acc, k) => ({ ...acc, [k]: [] }),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)